tests/util.py

```diff
@@ -47,7 +47,10 @@ def get_db_connection():
             pw = getpass("Enter the sql database password: ")
         else:
             pw = match.group(2)
-        db['connection'] = pymysql.connect(user=match.group(1), passwd=pw, host=match.group(3), db=match.group(4), port=3306, charset='utf8mb4')
```

Comment:
Remove before merging, just there for debugging

Reply:
Did you remove this?